Add documentation for sensor platform in bosch_alarm (#38347)
home-assistant/home-assistant.github.io: source/_integrations/bosch_alarm.markdown

The following {% term entities %} are provided:

- [Alarm Control Panel](#alarm-control-panel)
- [Sensor](#sensor)

### Alarm Control Panel

This integration adds an Alarm Control Panel device for each configured area, with the ability to issue arm/disarm commands.
This entity reports state (_disarmed_, _armed_away_, etc.).

## Sensor

> A sensor is provided for the panel containing the current faults from it.

Reviewer: ```suggestion
A sensor is provided for the panel to show the current error messages (faults).
```
Please correct me if I got this wrong. Added error message because it seems a more commonly used term to me.

> Three additional sensors are added per area, one containing information on if the panel is ready to arm, another containing information about current points that are faulting for this area, and another containing a list of alarms that have fired for the area.

Reviewer: ```suggestion
Three additional sensors are added per area:
- If the panel is ready to arm.
- A list of the current points that are faulting for this area.
- A list of alarms that have fired for the area.
```
